triton-vm/src/vm.rs

```diff
@@ -581,6 +594,32 @@ pub mod triton_vm_tests {
         )
     }
 
+    pub fn property_based_test_program_for_assert_vector() -> SourceCodeAndInput {
+        let mut rng = ThreadRng::default();
+        let st0 = rng.next_u64() % BFieldElement::QUOTIENT;
+        let st1 = rng.next_u64() % BFieldElement::QUOTIENT;
+        let st2 = rng.next_u64() % BFieldElement::QUOTIENT;
+        let st3 = rng.next_u64() % BFieldElement::QUOTIENT;
+        let st4 = rng.next_u64() % BFieldElement::QUOTIENT;
```

Comment:
Requires `use rand::Rng;`, but:

```suggestion
        let st0 = rng.gen_range(0..BFieldElement::QUOTIENT);
        let st1 = rng.gen_range(0..BFieldElement::QUOTIENT);
        let st2 = rng.gen_range(0..BFieldElement::QUOTIENT);
        let st3 = rng.gen_range(0..BFieldElement::QUOTIENT);
        let st4 = rng.gen_range(0..BFieldElement::QUOTIENT);
```